riva/clients/nmt/streaming_speech_translate_client.cc

```diff
@@ -302,12 +304,15 @@ StreamingSpeechTranslateClient::ReceiveResponses(
         std::cout << "ASR started... press `Ctrl-C' to stop recording\n\n";
         gotoxy(0, 5);
       }
-      std::cout << "result: " << result.DebugString();
+      VLOG(1) << "result: " << result.DebugString();
       if (print_transcripts_) {
         call->AppendResult(result);
       }
+      std::cout << "translated text: \"" << result.alternatives(0).transcript() << "\"" << std::endl;
+      result_file << result.alternatives(0).transcript() << std::endl;
```

Comment:
should add a check here if result_file is valid (say if user sets nmt_text_file="")

Reply:
Discussed offline with Viraj, this should be ok for now. Resolving.